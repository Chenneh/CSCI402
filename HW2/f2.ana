
==== Packets ====
p1: inter-arrival time should be 307.538ms

p2: inter-arrival time should be 269.628ms
    time in Q1 should be 105.940ms
    time in Q2 should be 0.014ms
    should request 342ms of service time
    time in S2 should be 358.622ms
    time in system should be 464.613ms

p3: inter-arrival time should be 209.092ms

p4: inter-arrival time should be 37.551ms

p5: inter-arrival time should be 284.903ms

p6: inter-arrival time should be 465.795ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.059ms
    should request 988ms of service time
    time in S2 should be 1030.272ms
    time in system should be 1030.351ms

p7: inter-arrival time should be 112.801ms

p8: inter-arrival time should be 151.985ms

p9: inter-arrival time should be 211.329ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.011ms
    should request 172ms of service time
    time in S1 should be 189.131ms
    time in system should be 189.162ms

p10: inter-arrival time should be 387.285ms
    time in Q1 should be 291.919ms
    time in Q2 should be 0.012ms
    should request 195ms of service time
    time in S1 should be 212.283ms
    time in system should be 504.230ms

p11: inter-arrival time should be 326.930ms

p12: inter-arrival time should be 196.429ms

p13: inter-arrival time should be 130.947ms

p14: inter-arrival time should be 365.598ms

p15: inter-arrival time should be 123.175ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.013ms
    should request 584ms of service time
    time in S1 should be 611.113ms
    time in system should be 611.143ms

p16: inter-arrival time should be 206.188ms

p17: inter-arrival time should be 370.079ms

p18: inter-arrival time should be 246.312ms

p19: inter-arrival time should be 163.873ms

p20: inter-arrival time should be 279.964ms

p21: inter-arrival time should be 332.412ms

p22: inter-arrival time should be 281.246ms
    time in Q1 should be 0.012ms
    time in Q2 should be 0.014ms
    should request 612ms of service time
    time in S1 should be 624.715ms
    time in system should be 624.755ms

p23: inter-arrival time should be 136.568ms
    time in Q1 should be 506.198ms
    time in Q2 should be 0.010ms
    should request 192ms of service time
    time in S2 should be 199.903ms
    time in system should be 706.126ms

p24: inter-arrival time should be 348.353ms

p25: inter-arrival time should be 157.877ms

p26: inter-arrival time should be 365.782ms

p27: inter-arrival time should be 462.604ms

p28: inter-arrival time should be 295.818ms

p29: inter-arrival time should be 148.572ms

p30: inter-arrival time should be 278.529ms

p31: inter-arrival time should be 122.752ms

p32: inter-arrival time should be 471.481ms

p33: inter-arrival time should be 429.941ms

p34: inter-arrival time should be 438.967ms

p35: inter-arrival time should be 253.436ms

p36: inter-arrival time should be 318.844ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.013ms
    should request 468ms of service time
    time in S2 should be 482.685ms
    time in system should be 482.716ms

p37: inter-arrival time should be 220.788ms
    time in Q1 should be 324.773ms
    time in Q2 should be 0.015ms
    should request 589ms of service time
    time in S1 should be 605.717ms
    time in system should be 930.521ms

p38: inter-arrival time should be 430.257ms

p39: inter-arrival time should be 166.658ms

p40: inter-arrival time should be 271.329ms

p41: inter-arrival time should be 433.847ms

p42: inter-arrival time should be 305.807ms

p43: inter-arrival time should be 407.084ms

p44: inter-arrival time should be 442.588ms

p45: inter-arrival time should be 161.709ms

p46: inter-arrival time should be 310.763ms

p47: inter-arrival time should be 149.305ms

p48: inter-arrival time should be 251.237ms

p49: inter-arrival time should be 181.016ms

p50: inter-arrival time should be 314.136ms

==== Statistics ====
average packet inter-arrival time = 0.27474216
average packet service time = 0.479382333333333
average number of packets in Q1 = 0.0857221125358825
average number of packets in Q2 = 1.12310006870443e-05
average number of packets in S1 = 0.156463814099454
average number of packets in S2 = 0.144501961274533
average time a packet spent in system = 0.615957444444444
standard deviation for time spent in system = 0.239025565871618
token drop probability = 0
packet drop probability = 0.82
