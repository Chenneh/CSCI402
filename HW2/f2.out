Emulation Parameters:
	number to arrive = 50
	r = 1.5
	B = 1
	tsfile = w2data/f2.txt
00000000.000ms: emulation begins
00000307.538ms: p1 arrives, needs 2 tokens, inter-arrival time = 307.538ms, dropped
00000577.166ms: p2 arrives, needs 1 tokens, inter-arrival time = 269.628ms
00000577.174ms: p2 enters Q1
00000683.093ms: token t1 arrives, token bucket now has 1 token 
00000683.114ms: p2 leaves Q1, time in Q1 = 105.938ms, token bucket now has 0 token 
00000683.136ms: p2 enters Q2
00000683.150ms: p2 leaves Q2, time in Q2 = 0.014ms
00000683.157ms: p2 begin service at S2, requesting 342ms of service
00000786.258ms: p3 arrives, needs 3 tokens, inter-arrival time = 209.092ms, dropped
00000823.809ms: p4 arrives, needs 5 tokens, inter-arrival time = 37.551ms, dropped
00001041.779ms: p2 departs from S2, servie time = 358.622ms, time in system = 464.613ms
00001108.712ms: p5 arrives, needs 3 tokens, inter-arrival time = 284.903ms, dropped
00001369.096ms: token t2 arrives, token bucket now has 1 token 
00001574.507ms: p6 arrives, needs 1 tokens, inter-arrival time = 465.795ms
00001574.516ms: p6 enters Q1
00001574.519ms: p6 leaves Q1, time in Q1 = 0.001ms, token bucket now has 0 token 
00001574.522ms: p6 enters Q2
00001574.581ms: p6 leaves Q2, time in Q2 = 0.059ms
00001574.586ms: p6 begin service at S2, requesting 988ms of service
00001687.308ms: p7 arrives, needs 3 tokens, inter-arrival time = 112.801ms, dropped
00001839.293ms: p8 arrives, needs 4 tokens, inter-arrival time = 151.985ms, dropped
00002050.607ms: token t3 arrives, token bucket now has 1 token 
00002050.622ms: p9 arrives, needs 1 tokens, inter-arrival time = 211.329ms
00002050.625ms: p9 enters Q1
00002050.628ms: p9 leaves Q1, time in Q1 = 0.001ms, token bucket now has 0 token 
00002050.631ms: p9 enters Q2
00002050.642ms: p9 leaves Q2, time in Q2 = 0.011ms
00002050.653ms: p9 begin service at S1, requesting 172ms of service
00002239.784ms: p9 departs from S1, servie time = 189.131ms, time in system = 189.162ms
00002437.907ms: p10 arrives, needs 1 tokens, inter-arrival time = 387.285ms
00002437.916ms: p10 enters Q1
00002604.858ms: p6 departs from S2, servie time = 1030.272ms, time in system = 1030.351ms
00002729.826ms: token t4 arrives, token bucket now has 1 token 
00002729.835ms: p10 leaves Q1, time in Q1 = 291.918ms, token bucket now has 0 token 
00002729.838ms: p10 enters Q2
00002729.850ms: p10 leaves Q2, time in Q2 = 0.012ms
00002729.854ms: p10 begin service at S1, requesting 195ms of service
00002764.837ms: p11 arrives, needs 5 tokens, inter-arrival time = 326.930ms, dropped
00002942.137ms: p10 departs from S1, servie time = 212.283ms, time in system = 504.230ms
00002961.266ms: p12 arrives, needs 4 tokens, inter-arrival time = 196.429ms, dropped
00003092.213ms: p13 arrives, needs 2 tokens, inter-arrival time = 130.947ms, dropped
00003413.722ms: token t5 arrives, token bucket now has 1 token 
00003457.811ms: p14 arrives, needs 4 tokens, inter-arrival time = 365.598ms, dropped
00003580.986ms: p15 arrives, needs 1 tokens, inter-arrival time = 123.175ms
00003580.994ms: p15 enters Q1
00003580.997ms: p15 leaves Q1, time in Q1 = 0.001ms, token bucket now has 0 token 
00003580.999ms: p15 enters Q2
00003581.012ms: p15 leaves Q2, time in Q2 = 0.013ms
00003581.016ms: p15 begin service at S1, requesting 584ms of service
00003787.174ms: p16 arrives, needs 3 tokens, inter-arrival time = 206.188ms, dropped
00004082.968ms: token t6 arrives, token bucket now has 1 token 
00004157.253ms: p17 arrives, needs 4 tokens, inter-arrival time = 370.079ms, dropped
00004192.129ms: p15 departs from S1, servie time = 611.113ms, time in system = 611.143ms
00004403.565ms: p18 arrives, needs 2 tokens, inter-arrival time = 246.312ms, dropped
00004567.438ms: p19 arrives, needs 4 tokens, inter-arrival time = 163.873ms, dropped
00004847.402ms: p20 arrives, needs 2 tokens, inter-arrival time = 279.964ms, dropped
00005179.814ms: p21 arrives, needs 3 tokens, inter-arrival time = 332.412ms, dropped
00005461.060ms: p22 arrives, needs 1 tokens, inter-arrival time = 281.246ms
00005461.068ms: p22 enters Q1
00005461.080ms: p22 leaves Q1, time in Q1 = 0.002ms, token bucket now has 0 token 
00005461.082ms: p22 enters Q2
00005461.096ms: p22 leaves Q2, time in Q2 = 0.014ms
00005461.100ms: p22 begin service at S1, requesting 612ms of service
00005597.628ms: p23 arrives, needs 1 tokens, inter-arrival time = 136.568ms
00005597.636ms: p23 enters Q1
00005945.981ms: p24 arrives, needs 5 tokens, inter-arrival time = 348.353ms, dropped
00006085.815ms: p22 departs from S1, servie time = 624.715ms, time in system = 624.755ms
00006103.828ms: token t11 arrives, token bucket now has 1 token 
00006103.834ms: p23 leaves Q1, time in Q1 = 506.196ms, token bucket now has 0 token 
00006103.837ms: p23 enters Q2
00006103.847ms: p23 leaves Q2, time in Q2 = 0.010ms
00006103.851ms: p23 begin service at S2, requesting 192ms of service
00006103.858ms: p25 arrives, needs 4 tokens, inter-arrival time = 157.877ms, dropped
00006303.754ms: p23 departs from S2, servie time = 199.903ms, time in system = 706.126ms
00006469.640ms: p26 arrives, needs 4 tokens, inter-arrival time = 365.782ms, dropped
00006809.997ms: token t12 arrives, token bucket now has 1 token 
00006932.244ms: p27 arrives, needs 4 tokens, inter-arrival time = 462.604ms, dropped
00007228.062ms: p28 arrives, needs 2 tokens, inter-arrival time = 295.818ms, dropped
00007376.634ms: p29 arrives, needs 5 tokens, inter-arrival time = 148.572ms, dropped
00007655.163ms: p30 arrives, needs 2 tokens, inter-arrival time = 278.529ms, dropped
00007777.915ms: p31 arrives, needs 4 tokens, inter-arrival time = 122.752ms, dropped
00008249.396ms: p32 arrives, needs 2 tokens, inter-arrival time = 471.481ms, dropped
00008679.337ms: p33 arrives, needs 2 tokens, inter-arrival time = 429.941ms, dropped
00009118.304ms: p34 arrives, needs 2 tokens, inter-arrival time = 438.967ms, dropped
00009371.740ms: p35 arrives, needs 5 tokens, inter-arrival time = 253.436ms, dropped
00009690.584ms: p36 arrives, needs 1 tokens, inter-arrival time = 318.844ms
00009690.592ms: p36 enters Q1
00009690.595ms: p36 leaves Q1, time in Q1 = 0.001ms, token bucket now has 0 token 
00009690.598ms: p36 enters Q2
00009690.611ms: p36 leaves Q2, time in Q2 = 0.013ms
00009690.615ms: p36 begin service at S2, requesting 468ms of service
00009911.372ms: p37 arrives, needs 1 tokens, inter-arrival time = 220.788ms
00009911.380ms: p37 enters Q1
00010173.300ms: p36 departs from S2, servie time = 482.685ms, time in system = 482.716ms
00010236.143ms: token t21 arrives, token bucket now has 1 token 
00010236.153ms: p37 leaves Q1, time in Q1 = 324.771ms, token bucket now has 0 token 
00010236.156ms: p37 enters Q2
00010236.171ms: p37 leaves Q2, time in Q2 = 0.015ms
00010236.176ms: p37 begin service at S1, requesting 589ms of service
00010341.629ms: p38 arrives, needs 3 tokens, inter-arrival time = 430.257ms, dropped
00010508.287ms: p39 arrives, needs 3 tokens, inter-arrival time = 166.658ms, dropped
00010779.616ms: p40 arrives, needs 4 tokens, inter-arrival time = 271.329ms, dropped
00010841.893ms: p37 departs from S1, servie time = 605.717ms, time in system = 930.521ms
00010909.386ms: token t22 arrives, token bucket now has 1 token 
00011213.463ms: p41 arrives, needs 2 tokens, inter-arrival time = 433.847ms, dropped
00011519.270ms: p42 arrives, needs 5 tokens, inter-arrival time = 305.807ms, dropped
00011926.354ms: p43 arrives, needs 5 tokens, inter-arrival time = 407.084ms, dropped
00012368.942ms: p44 arrives, needs 5 tokens, inter-arrival time = 442.588ms, dropped
00012530.651ms: p45 arrives, needs 4 tokens, inter-arrival time = 161.709ms, dropped
00012841.414ms: p46 arrives, needs 3 tokens, inter-arrival time = 310.763ms, dropped
00012990.719ms: p47 arrives, needs 5 tokens, inter-arrival time = 149.305ms, dropped
00013241.956ms: p48 arrives, needs 5 tokens, inter-arrival time = 251.237ms, dropped
00013422.972ms: p49 arrives, needs 4 tokens, inter-arrival time = 181.016ms, dropped
00013737.108ms: p50 arrives, needs 4 tokens, inter-arrival time = 314.136ms, dropped
00014335.321ms: emulation ends
	average packet inter-arrival time time = 0.274742
	average packet service time = 0.479382

	average number of packets in Q1 = 0.0857203
	average number of packets in Q2 = 1.1231e-05
	average number of packets in S1 = 0.156464
	average number of packets in S2 = 0.144502
	average time a packet spent in system = 0.615957
	standard deviation for time spent in system = 0.239026

	token drop probability = 0
	packet drop probability = 0.82

