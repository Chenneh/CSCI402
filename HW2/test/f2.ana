
==== Packets ====
p1: inter-arrival time should be 297.611ms

p2: inter-arrival time should be 268.504ms
    time in Q1 should be 117.093ms
    time in Q2 should be 0.014ms
    should request 342ms of service time
    time in S2 should be 342.935ms
    time in system should be 460.080ms

p3: inter-arrival time should be 209.155ms

p4: inter-arrival time should be 61.433ms

p5: inter-arrival time should be 295.460ms

p6: inter-arrival time should be 490.393ms
    time in Q1 should be 0.002ms
    time in Q2 should be 0.014ms
    should request 988ms of service time
    time in S2 should be 1004.737ms
    time in system should be 1004.769ms

p7: inter-arrival time should be 110.290ms

p8: inter-arrival time should be 136.387ms

p9: inter-arrival time should be 225.356ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.014ms
    should request 172ms of service time
    time in S1 should be 179.560ms
    time in system should be 179.597ms

p10: inter-arrival time should be 380.326ms
    time in Q1 should be 237.379ms
    time in Q2 should be 0.013ms
    should request 195ms of service time
    time in S1 should be 211.841ms
    time in system should be 449.250ms

p11: inter-arrival time should be 336.971ms

p12: inter-arrival time should be 206.746ms

p13: inter-arrival time should be 119.384ms

p14: inter-arrival time should be 367.683ms

p15: inter-arrival time should be 124.901ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.014ms
    should request 584ms of service time
    time in S1 should be 591.781ms
    time in system should be 591.813ms

p16: inter-arrival time should be 207.886ms

p17: inter-arrival time should be 364.166ms

p18: inter-arrival time should be 250.123ms

p19: inter-arrival time should be 156.508ms

p20: inter-arrival time should be 278.570ms

p21: inter-arrival time should be 341.933ms

p22: inter-arrival time should be 283.339ms
    time in Q1 should be 0.002ms
    time in Q2 should be 0.014ms
    should request 612ms of service time
    time in S1 should be 619.190ms
    time in system should be 619.221ms

p23: inter-arrival time should be 151.871ms
    time in Q1 should be 447.303ms
    time in Q2 should be 0.013ms
    should request 192ms of service time
    time in S2 should be 200.282ms
    time in system should be 647.614ms

p24: inter-arrival time should be 343.058ms

p25: inter-arrival time should be 158.108ms

p26: inter-arrival time should be 363.404ms

p27: inter-arrival time should be 453.153ms

p28: inter-arrival time should be 286.435ms

p29: inter-arrival time should be 154.894ms

p30: inter-arrival time should be 268.521ms

p31: inter-arrival time should be 123.052ms

p32: inter-arrival time should be 468.114ms

p33: inter-arrival time should be 430.821ms

p34: inter-arrival time should be 437.960ms

p35: inter-arrival time should be 262.281ms

p36: inter-arrival time should be 320.278ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.019ms
    should request 468ms of service time
    time in S2 should be 479.128ms
    time in system should be 479.170ms

p37: inter-arrival time should be 221.796ms
    time in Q1 should be 216.871ms
    time in Q2 should be 0.009ms
    should request 589ms of service time
    time in S1 should be 605.906ms
    time in system should be 822.802ms

p38: inter-arrival time should be 444.720ms

p39: inter-arrival time should be 170.437ms

p40: inter-arrival time should be 271.173ms

p41: inter-arrival time should be 445.723ms

p42: inter-arrival time should be 298.821ms

p43: inter-arrival time should be 421.011ms

p44: inter-arrival time should be 442.575ms

p45: inter-arrival time should be 162.834ms

p46: inter-arrival time should be 322.850ms

p47: inter-arrival time should be 148.638ms

p48: inter-arrival time should be 260.431ms

p49: inter-arrival time should be 183.313ms

p50: inter-arrival time should be 314.693ms

==== Statistics ====
average packet inter-arrival time = 0.2768818
average packet service time = 0.470595555555556
average number of packets in Q1 = 0.071528728117188
average number of packets in Q2 = 8.70709657160179e-06
average number of packets in S1 = 0.155062014539868
average number of packets in S2 = 0.142338699456094
average time a packet spent in system = 0.583812888888889
standard deviation for time spent in system = 0.222976276387544
token drop probability = 0.67741935483871
packet drop probability = 0.82
