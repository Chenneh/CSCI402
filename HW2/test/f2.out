Emulation Parameters:
	number to arrive = 50
	r = 1.5
	B = 1
	tsfile = w2data/f2.txt
00000000.000ms: emulation begins
00000297.611ms: p1 arrives, needs 2 tokens, inter-arrival time = 297.611ms, dropped
00000566.115ms: p2 arrives, needs 1 tokens, inter-arrival time = 268.504ms
00000566.123ms: p2 enters Q1
00000683.197ms: token t1 arrives, token bucket now has 1 token 
00000683.216ms: p2 leaves Q1, time in Q1 = 117.093ms, token bucket now has 0 token 
00000683.238ms: p2 enters Q2
00000683.252ms: p2 leaves Q2, time in Q2 = 0.014ms
00000683.260ms: p2 begin service at S2, requesting 342ms of service
00000775.270ms: p3 arrives, needs 3 tokens, inter-arrival time = 209.155ms, dropped
00000836.703ms: p4 arrives, needs 5 tokens, inter-arrival time = 61.433ms, dropped
00001026.195ms: p2 departs from S2, servie time = 342.935ms, time in system = 460.080ms
00001132.163ms: p5 arrives, needs 3 tokens, inter-arrival time = 295.460ms, dropped
00001362.263ms: token t2 arrives, token bucket now has 1 token 
00001622.556ms: p6 arrives, needs 1 tokens, inter-arrival time = 490.393ms
00001622.565ms: p6 enters Q1
00001622.567ms: p6 leaves Q1, time in Q1 = 0.002ms, token bucket now has 0 token 
00001622.570ms: p6 enters Q2
00001622.584ms: p6 leaves Q2, time in Q2 = 0.014ms
00001622.588ms: p6 begin service at S2, requesting 988ms of service
00001732.846ms: p7 arrives, needs 3 tokens, inter-arrival time = 110.290ms, dropped
00001869.233ms: p8 arrives, needs 4 tokens, inter-arrival time = 136.387ms, dropped
00002029.518ms: token t3 arrives, token bucket now has 1 token 
00002094.589ms: p9 arrives, needs 1 tokens, inter-arrival time = 225.356ms
00002094.597ms: p9 enters Q1
00002094.600ms: p9 leaves Q1, time in Q1 = 0.003ms, token bucket now has 0 token 
00002094.602ms: p9 enters Q2
00002094.616ms: p9 leaves Q2, time in Q2 = 0.014ms
00002094.626ms: p9 begin service at S1, requesting 172ms of service
00002274.186ms: p9 departs from S1, servie time = 179.560ms, time in system = 179.597ms
00002474.915ms: p10 arrives, needs 1 tokens, inter-arrival time = 380.326ms
00002474.924ms: p10 enters Q1
00002627.325ms: p6 departs from S2, servie time = 1004.737ms, time in system = 1004.769ms
00002712.295ms: token t4 arrives, token bucket now has 1 token 
00002712.303ms: p10 leaves Q1, time in Q1 = 237.379ms, token bucket now has 0 token 
00002712.307ms: p10 enters Q2
00002712.320ms: p10 leaves Q2, time in Q2 = 0.013ms
00002712.324ms: p10 begin service at S1, requesting 195ms of service
00002811.886ms: p11 arrives, needs 5 tokens, inter-arrival time = 336.971ms, dropped
00002924.165ms: p10 departs from S1, servie time = 211.841ms, time in system = 449.250ms
00003018.632ms: p12 arrives, needs 4 tokens, inter-arrival time = 206.746ms, dropped
00003138.016ms: p13 arrives, needs 2 tokens, inter-arrival time = 119.384ms, dropped
00003381.861ms: token t5 arrives, token bucket now has 1 token 
00003505.699ms: p14 arrives, needs 4 tokens, inter-arrival time = 367.683ms, dropped
00003630.600ms: p15 arrives, needs 1 tokens, inter-arrival time = 124.901ms
00003630.608ms: p15 enters Q1
00003630.611ms: p15 leaves Q1, time in Q1 = 0.003ms, token bucket now has 0 token 
00003630.614ms: p15 enters Q2
00003630.628ms: p15 leaves Q2, time in Q2 = 0.014ms
00003630.632ms: p15 begin service at S1, requesting 584ms of service
00003838.486ms: p16 arrives, needs 3 tokens, inter-arrival time = 207.886ms, dropped
00004065.374ms: token t6 arrives, token bucket now has 1 token 
00004202.652ms: p17 arrives, needs 4 tokens, inter-arrival time = 364.166ms, dropped
00004222.413ms: p15 departs from S1, servie time = 591.781ms, time in system = 591.813ms
00004452.775ms: p18 arrives, needs 2 tokens, inter-arrival time = 250.123ms, dropped
00004609.283ms: p19 arrives, needs 4 tokens, inter-arrival time = 156.508ms, dropped
00004065.382ms: token t7 arrives, dropped
00004746.583ms: token t8 arrives, dropped
00004887.853ms: p20 arrives, needs 2 tokens, inter-arrival time = 278.570ms, dropped
00005229.786ms: p21 arrives, needs 3 tokens, inter-arrival time = 341.933ms, dropped
00004746.586ms: token t9 arrives, dropped
00005429.769ms: token t10 arrives, dropped
00005513.125ms: p22 arrives, needs 1 tokens, inter-arrival time = 283.339ms
00005513.133ms: p22 enters Q1
00005513.135ms: p22 leaves Q1, time in Q1 = 0.002ms, token bucket now has 0 token 
00005513.138ms: p22 enters Q2
00005513.152ms: p22 leaves Q2, time in Q2 = 0.014ms
00005513.156ms: p22 begin service at S1, requesting 612ms of service
00005664.996ms: p23 arrives, needs 1 tokens, inter-arrival time = 151.871ms
00005665.006ms: p23 enters Q1
00006008.054ms: p24 arrives, needs 5 tokens, inter-arrival time = 343.058ms, dropped
00006112.301ms: token t11 arrives, token bucket now has 1 token 
00006112.309ms: p23 leaves Q1, time in Q1 = 447.303ms, token bucket now has 0 token 
00006112.312ms: p23 enters Q2
00006112.325ms: p23 leaves Q2, time in Q2 = 0.013ms
00006112.328ms: p23 begin service at S2, requesting 192ms of service
00006132.346ms: p22 departs from S1, servie time = 619.190ms, time in system = 619.221ms
00006166.162ms: p25 arrives, needs 4 tokens, inter-arrival time = 158.108ms, dropped
00006312.610ms: p23 departs from S2, servie time = 200.282ms, time in system = 647.614ms
00006529.566ms: p26 arrives, needs 4 tokens, inter-arrival time = 363.404ms, dropped
00006794.951ms: token t12 arrives, token bucket now has 1 token 
00006982.719ms: p27 arrives, needs 4 tokens, inter-arrival time = 453.153ms, dropped
00007269.154ms: p28 arrives, needs 2 tokens, inter-arrival time = 286.435ms, dropped
00007424.048ms: p29 arrives, needs 5 tokens, inter-arrival time = 154.894ms, dropped
00006794.959ms: token t13 arrives, dropped
00007466.037ms: token t14 arrives, dropped
00007692.569ms: p30 arrives, needs 2 tokens, inter-arrival time = 268.521ms, dropped
00007815.621ms: p31 arrives, needs 4 tokens, inter-arrival time = 123.052ms, dropped
00007466.039ms: token t15 arrives, dropped
00008138.460ms: token t16 arrives, dropped
00008283.735ms: p32 arrives, needs 2 tokens, inter-arrival time = 468.114ms, dropped
00008714.556ms: p33 arrives, needs 2 tokens, inter-arrival time = 430.821ms, dropped
00008138.462ms: token t17 arrives, dropped
00008820.100ms: token t18 arrives, dropped
00009152.516ms: p34 arrives, needs 2 tokens, inter-arrival time = 437.960ms, dropped
00009414.797ms: p35 arrives, needs 5 tokens, inter-arrival time = 262.281ms, dropped
00008820.103ms: token t19 arrives, dropped
00009496.030ms: token t20 arrives, dropped
00009735.075ms: p36 arrives, needs 1 tokens, inter-arrival time = 320.278ms
00009735.085ms: p36 enters Q1
00009735.088ms: p36 leaves Q1, time in Q1 = 0.003ms, token bucket now has 0 token 
00009735.092ms: p36 enters Q2
00009735.111ms: p36 leaves Q2, time in Q2 = 0.019ms
00009735.117ms: p36 begin service at S2, requesting 468ms of service
00009956.871ms: p37 arrives, needs 1 tokens, inter-arrival time = 221.796ms
00009956.880ms: p37 enters Q1
00010173.744ms: token t21 arrives, token bucket now has 1 token 
00010173.751ms: p37 leaves Q1, time in Q1 = 216.871ms, token bucket now has 0 token 
00010173.755ms: p37 enters Q2
00010173.764ms: p37 leaves Q2, time in Q2 = 0.009ms
00010173.767ms: p37 begin service at S1, requesting 589ms of service
00010214.245ms: p36 departs from S2, servie time = 479.128ms, time in system = 479.170ms
00010401.591ms: p38 arrives, needs 3 tokens, inter-arrival time = 444.720ms, dropped
00010572.028ms: p39 arrives, needs 3 tokens, inter-arrival time = 170.437ms, dropped
00010779.673ms: p37 departs from S1, servie time = 605.906ms, time in system = 822.802ms
00010843.185ms: token t22 arrives, token bucket now has 1 token 
00010843.201ms: p40 arrives, needs 4 tokens, inter-arrival time = 271.173ms, dropped
00011288.924ms: p41 arrives, needs 2 tokens, inter-arrival time = 445.723ms, dropped
00010843.194ms: token t23 arrives, dropped
00011528.104ms: token t24 arrives, dropped
00011587.745ms: p42 arrives, needs 5 tokens, inter-arrival time = 298.821ms, dropped
00012008.756ms: p43 arrives, needs 5 tokens, inter-arrival time = 421.011ms, dropped
00011528.107ms: token t25 arrives, dropped
00012197.931ms: token t26 arrives, dropped
00012451.331ms: p44 arrives, needs 5 tokens, inter-arrival time = 442.575ms, dropped
00012614.165ms: p45 arrives, needs 4 tokens, inter-arrival time = 162.834ms, dropped
00012197.934ms: token t27 arrives, dropped
00012876.743ms: token t28 arrives, dropped
00012937.015ms: p46 arrives, needs 3 tokens, inter-arrival time = 322.850ms, dropped
00013085.653ms: p47 arrives, needs 5 tokens, inter-arrival time = 148.638ms, dropped
00013346.084ms: p48 arrives, needs 5 tokens, inter-arrival time = 260.431ms, dropped
00013529.397ms: p49 arrives, needs 4 tokens, inter-arrival time = 183.313ms, dropped
00012876.745ms: token t29 arrives, dropped
00013559.513ms: token t30 arrives, dropped
00013844.090ms: p50 arrives, needs 4 tokens, inter-arrival time = 314.693ms, dropped
00013559.515ms: token t31 arrives, dropped
00014241.257ms: emulation ends

Statistics:

	average packet inter-arrival time time = 0.2768818
	average packet service time = 0.47059556

	average number of packets in Q1 = 0.071528728
	average number of packets in Q2 = 8.7070765e-06
	average number of packets in S1 = 0.15506201
	average number of packets in S2 = 0.1423387
	average time a packet spent in system = 0.58381289
	standard deviation for time spent in system = 0.22297628

	token drop probability = 0.67741935
	packet drop probability = 0.82

